tutorials/monitoring/4-endpoint-performance/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Monitoring NServiceBus solutions: Throughput and processing time"
+reviewed: 2017-10-10
+summary: Measuring individual endpoint performance with throughput and processing time.
+extensions:
+- !!tutorial
+  nextText: "Next Lesson: Queue length and critical time"
+  nextUrl: tutorials/monitoring/5-cross-endpoint-performance
+---
+
+include: monitoring-intro-paragraph
+
+This fourth lesson guides you through the throughput and processing time metrics and teaches you how to use those metrics to interpret the performance of an individual endpoint. Examples are shown using the sample monitoring solution.
```

Comment:
I think we should add an information that we assume that users have configured monitoring in the sample solution as described in lesson 3.